python/ray/serve/model_wrappers.py

```diff
@@ -69,13 +124,15 @@ def collate_dataframe(
     def unpack(output_df):
         if isinstance(output_df, list):
             return output_df
-        assert isinstance(
-            output_df, pd.DataFrame
-        ), f"The output should be a Pandas DataFrame but Serve got {type(output_df)}"
-        assert len(output_df) % batch_size == 0, (
-            f"The output dataframe should have length divisible by {batch_size}, "
-            f"but Serve got length {len(output_df)}."
-        )
+        if not isinstance(output_df, pd.DataFrame):
+            raise TypeError(
+                f"The output should be a Pandas DataFrame but Serve got {type(output_df)}"
+            )
+        if len(output_df) % batch_size != 0:
+            raise ValueError(
+                f"The output dataframe should have length divisible by {batch_size}, "
```

Comment:
The phrase `divisible by {batch_size}` might be confusing for users since it'll change whenever the input batch size changes. The error might look like a moving target for users.

Could we add more context to that phrase and explain where the `batch_size` number comes from?